.gitignore

```diff
@@ -1,10 +1,4 @@
-v7
```

Comment:
On unices this is the name of the v7 executable.
Next line is also a build product.